lib/matplotlib/font_manager.py

```diff
@@ -929,6 +940,45 @@ def set_fontconfig_pattern(self, pattern):
             else:
                 getattr(self, "set_" + key)(val)
 
+    def get_math_fontfamily(self):
+        """
+        Return the name of the font family used for math text.
+        The default font is :rc:`mathtext.fontset`.
+        """
+
+        if self._math_fontfamily is None:
+            return rcParams['mathtext.fontset']
+        return self._math_fontfamily
+
+    def set_math_fontfamily(self, fontset):
+        """
+        The *math_fontfamily* is the family of fonts used in case this text is
+        rendered in math mode. If no *math_fontfamily* is defined, the
+        fontset defined in :rc:`mathtext.fontset` will be used.
+
+        Parameters
+        ----------
+        fontset : str, the name of the fontset. Available fontsets are defined
+        in the matplotlibrc.template file :ref:`here
+        <customizing-with-matplotlibrc-files>`
+
+        See Also
+        --------
+        .text.Text.get_math_fontfamily
+        """
+        if fontset is None:
+            self._math_fontfamily = None
+            return
+        # here we validate the parameter fontset just as if it were
+        # passed to rcParams['mathtext.fontset'], but with a customized text
+        valid_fonts = _validators['mathtext.fontset'].valid.values()
+        if fontset not in valid_fonts:
+            msg = (f"Parameter math_fontfamily ({fontset}) must be one of "
+                   f"the following: {[*valid_fonts]}")
+            raise ValueError (msg)
```

Comment:
We have a helper function for this:
```suggestion
        cbook._check_in_list(valid_fonts, math_fontfamily=math_fontfamily)
```